arcor2_AREditor/Assets/TABLET/Scripts/GUI/WalktroughManager.cs

```diff
@@ -0,0 +1,489 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using UnityEngine;
+
+public class WalktroughManager : MonoBehaviour {
+
+    [SerializeField]
+    public static WalktroughManager Instance {
+        get; private set;
+    }
+    public List<GameObject> Buttons;
```

Comment:
Again, I would suggest using `OnValidate` - just to simply check whether it is set.